default.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs ? import ./pkgs.nix }:
+
+with pkgs;
+
+let
+  hex = fetchurl {
```

Comment:
Maybe we should add `hex` to `serokell-closure`?

Reply:
Probably a good idea, yes. Can you do that?

Reply:
Actually, why not submit upstream to nixpkgs?